platforms/aws/giantnetes/backend.tf

```diff
@@ -0,0 +1 @@
+../../../build/backend.tf
```

Comment:
same as for Azure. Symlink will point to cluster-specific tf state backend configuration